admin/add-ftp-page.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require('../template/top.php');
+head('Add FTP user', true);
+?>
+<style>
+    #contact-form-submit-area {
+        margin-top: 10px;
+    }
+
+    #contact-form-submit-area > span > * {
+        display: inline-block;
+        vertical-align: middle;
+    }
+</style>
+
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+
+<main class="page-content">
+    <section class="section-50 section-md-75 section-md-100 section-lg-120 section-xl-150 bg-wild-sand">
+        <div class="shell text-left">
+            <h2>just fill in the form to add FTP user</h2>
+            <form action="../ajax/add-ftp-user.php" method="post">
```

Comment:
I see now that this isn't actually doing AJAX at all! So, you can move all of your code from the AJAX to the top of this file and the action can just be `action=""` and it'll always POST to itself.

Reply:
moved it to the top of the file